README.md

```diff
@@ -7,3 +7,8 @@ A collection of messages, reminders and questions that need to be socialized wit
 2. Start the app `uvicorn main:app`
 3. To check if the app started, in a new terminal `curl http://127.0.0.1:8000/message`
 4. If all goes well, you should see a message displayed
+
+### How to run test
+
+1. Run the test `coverage run -m pytest tests/test_main.py` to test the test cases.
```

Comment:
Can you add instructions for the remaining unit tests as well? Just `coverage run -m pytest` should do the trick. The current command runs only the API test.  

Side note: We rarely run coverage reports against API tests. Coverage usually applies to unit tests.

Reply:
@qxf2 , I have resolved this also kept the side note as well there. In case you don't like it I will remove the same.